pkg/plugins/manager/loader/angulardetector/inspector.go

```diff
@@ -0,0 +1,45 @@
+package angulardetector
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/grafana/grafana/pkg/plugins"
+)
+
+// Inspector can inspect a plugin and determine if it's an Angular plugin or not.
+type Inspector interface {
+	// Inspect takes a plugin and checks if the plugin is using Angular.
+	Inspect(ctx context.Context, p *plugins.Plugin) (bool, error)
```

Comment:
can you use different packages for the "inspector" and the "detector"? it will help for the readability and the separation of concerns.

Reply:
:+1: I agree it makes sense to separate those